client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -53,7 +54,8 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
   private val pushReplicateEnabled = conf.pushReplicateEnabled
   private val partitionSplitThreshold = conf.partitionSplitThreshold
   private val partitionSplitMode = conf.partitionSplitMode
-  private val partitionType = conf.shufflePartitionType
+  // shuffle id -> partition type
+  private val shufflePartitionType = new ConcurrentHashMap[Int, PartitionType]()
```

Comment:
This is for the future? Will an application have both map partition and reduce partition?

Reply:
I don't think we should limit this. May be add this in future